sunpy/net/helioviewer.py

```diff
@@ -450,3 +499,17 @@ def _request(self, params):
     def _format_date(self, date):
         """Formats a date for Helioviewer API requests"""
         return parse_time(date).isot + "Z"
+
+    def _get_source_id(self, source_id, key):
+        if source_id is None:
+            try:
+                source_id_list = list(partial_match(key,self.data_sources))
+                if len(source_id_list) == 1:
+                    source_id = source_id_list[0]
+                else:
+                    source_id = self.data_sources[key]
```

Comment:
Do we need to do this anymore?

Reply:
We can replace it with the list of values returned by the function